src/Native/Bootstrap/platform.unix.cpp

```diff
@@ -70,4 +70,24 @@ extern "C"
     {
         throw "RhCompatibleReentrantWaitAny";
     }
+
+    void EnumDynamicTimeZoneInformation()
```

Comment:
It was a problem on Unix only. I have not really drilled into why it was broken. Also, it was broken since not too long ago - maybe one month.